src/netron_output.cpp

```diff
@@ -57,7 +57,8 @@ int get_onnx_type(shape::type_t s_type)
     case shape::fp8e5m2_type: return 19;
     case shape::fp8e5m2fnuz_type: return 20;
     case shape::tuple_type: return 0;
-    case shape::fp4x2_type: return 21; // TODO update this when the type is added
+    case shape::fp4x2_type: return 23;
```

Comment:
Not sure why this is no 23 from 21? Shouldn't these be the same and not change or is this more related to some other representation of things?